.gitignore

```diff
@@ -55,3 +55,6 @@ target
 **/aggregated-translations/*.*
 lib
 props-table
+**/__snapshots__/screen
+**/__snapshots__/diff
+errorShots
```

Comment:
This was updated to [errorScreenshots ](https://github.com/cerner/terra-toolkit/blob/master/src/wdio/conf.js#L22)